sympy/physics/control/lti.py

```diff
@@ -0,0 +1,122 @@
+from sympy import (
+    Basic, Matrix, latex, ShapeError, Mul,
+    together, evaluate, expand, cancel,
+    ImmutableMatrix, MutableMatrix, MutableDenseMatrix,
+)
```

Comment:
```suggestion
from sympy import (
    Basic, Matrix, ShapeError, Mul,
    ImmutableMatrix, MutableMatrix, MutableDenseMatrix,
)
```
I think that the first thing is making the tests pass
You should check the flake8 log
https://travis-ci.org/github/sympy/sympy/jobs/691766576

Reply:
There may not even be use of importing all the matrix types, and you'll always get missing ones like sparse matrix types.
It is sufficient to check with `MatrixBase`

Reply:
I don't think we're quite there yet. I've asked @namannimmo10 to write only unit tests for review for now (no implementation code).